packs/hello-universe-1.1.2/values.yaml

```diff
@@ -0,0 +1,16 @@
+# spectrocloud.com/enabled-presets: Backend:disable-api
+
+pack:
+  content:
+    images:
+      - image: ghcr.io/spectrocloud/hello-universe:1.1.2
+
+
+manifests:
+  hello-universe:
+    images:
+      hellouniverse: ghcr.io/spectrocloud/hello-universe:1.1.2
```

Comment:
If the API is enabled, can we also expose the API image and DB image. That way we can test new configurations more easily in the future without always requiring a pack change 😄

Reply:
Added new schema placeholders for this. Note that the default preset is the API disabled configuration, so this will only show up in `presets.yaml`.